IntelliTect.TestTools.Selenate/GoogleSearch/ExampleTests.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using IntelliTect.TestTools.Selenate;
+
+namespace GoogleSearch
+{
+    [TestClass]
+    public class ExampleTests
+    {
+        [TestInitialize]
+        public void Setup()
+        {
+            Browser = new GoogleBrowser(BrowserType.Chrome);
+            Google = new GoogleFunctions(Browser);
+            Harness = new GoogleHarness(Browser);
+        }
+
+        [TestMethod]
+        public void SearchForSeleniumOnGoogle()
```

Comment:
I would expect these unit tests to become async methods since the code they are calling should be async methods.
```C#
[TestMethod]
public async Task SearchForSeleniumOnGoogle()
```